### ISSUE TITLE
Consider tweaking default fullsweep_after

### ISSUE THREAD

**Issue by michaelklishin** [rabbitmq/rabbitmq-server#629]
See [this mailing list report by Scott Nichol](https://groups.google.com/d/msg/rabbitmq-users/6ucb0Dwns-M/uMoM1UuuDAAJ).

**Comment by michaelklishin** [comment]
@dcorbacho @Dzol @Ayanda-D @dumbbell @Gsantomaggio do you have experience with low `ERL_FULLSWEEP_AFTER` values?

@bdshroyer FYI as you expressed interest in how the runtime works.

**Comment by michaelklishin** [comment]
Doh, forgot @essen.

**Comment by essen** [comment]
I have no experience with this but this sounds like a good task for or with @gerhard :-)

**Comment by gerhard** [comment]
Yes, it would make sense to try a different default value and see how it impacts message throughput & latency.

@Vanlightly I think it's time we finished https://www.pivotaltracker.com/story/show/169980927 and used the resulting setup to find the sweet spot for this. Are you up for it next week?
